source/module_base/global_variable.cpp

```diff
@@ -296,6 +296,7 @@ double sc_thr = 1.0e-6;
 int nsc = 100;
 int nsc_min = 2;
 int sc_scf_nmin = 2;
+double sc_scf_start = 0.1;
```

Comment:
do not add any GlobalV, use an alternative method.

Reply:
@mohanchen Thanks for your comments. I will make a new PR to achieve this. So close this one now.